.github/ISSUE_TEMPLATE/improve-existing-docs.yaml

```diff
@@ -0,0 +1,47 @@
+name: Improve existing content
+description: Make a suggestion to improve the content in an existing article
+title: "['Improve existing content']: "
+labels:
+  - content
+body:
+  # HUBBERS BEWARE! THE GITHUB/DOCS REPO IS PUBLIC TO THE ENTIRE INTERNET. OPEN AN ISSUE IN GITHUB/DOCS-CONTENT INSTEAD.
+  - type: markdown
+    attributes:
+      value: |
+        For questions, ask in [Discussions](https://github.com/github/docs/discussions)
+        Before you file an issue read the
+        - [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)
+        - [Contributing guide](https://github.com/github/docs/blob/main/CONTRIBUTING.md)
+
+  - type: checkboxes
```

Comment:
I feel like an "I agree to comply with the Code of Conduct" checkbox is going to be more useful here than a "I have searched the existing issues" checkbox. There's no cost to the issue filer for checking and not complying and no benefit to us saying, "But you said you'd do it!". Whereas the Code of Conduct checkbox is an agreement to be bound by certain rules that include penalties for not following the rules.